views/admin/blocks/wd_admin_payment_main_form.tpl

```diff
@@ -135,7 +135,11 @@
     <tr>
       <td colspan="2">
         <div class="wdoxidee-messagebox wdoxidee-messagebox--error">
-          [{oxmultilang ident="wd_error_save_failed"}]
+          [{if $paymentMethod->getName() === 'wdcreditcard'}]
```

Comment:
So on every save error if the payment method is credit card we show the mismatch url error? Is that correct?

Reply:
yes

Reply:
Can we optimize and check on some code or so? Else DB errors will result in an mismatch error.

Reply:
Good catch, we will have a look.